pkg/awsutils/imds_test.go

```diff
@@ -214,30 +214,14 @@ func TestGetVPCIPv4CIDRBlocks(t *testing.T) {
 
 func TestGetVPCIPv6CIDRBlocks(t *testing.T) {
 	f := TypedIMDS{FakeIMDS(map[string]interface{}{
-		"network/interfaces/macs/02:c5:f8:3e:6b:27/vpc-ipv6-cidr-blocks": "2001:db8::/64",
+		"network/interfaces/macs/02:c5:f8:3e:6b:27/subnet-ipv6-cidr-blocks": "2001:db8::/64",
 	})}
 
 	ips, err := f.GetVPCIPv6CIDRBlocks(context.TODO(), "02:c5:f8:3e:6b:27")
 	if assert.NoError(t, err) {
-		assert.Equal(t, ips, []net.IPNet{{IP: net.ParseIP("2001:db8::"), Mask: net.CIDRMask(64, 128)}})
-	}
-
-	nov6 := TypedIMDS{FakeIMDS(map[string]interface{}{
```

Comment:
Yeah, I'll add it back. I initially removed it because I observed that for `404` scenarios, SDK was hanging forever (repeatedly retrying) and so I thought this test is not accurately depicting the SDK behavior. More of a check to add it back once the issue was fixed, and I think they recently addressed it prior to PD.